.github/workflows/avail-light-client.yml

```diff
@@ -1,4 +1,4 @@
-name: Releaser
+name: Avail Light Release
```

Comment:
Should we follow the full name of "Avail Light Client"?

Reply:
Done